src/Chart/createChart.jsx

```diff
@@ -0,0 +1,127 @@
+import React, {Component, PropTypes} from 'react'
+import 'chart.js'
+
+/* global Chart */
+
+export default function createChart (ChartElement) {
+  class ChartComponent extends Component {
+    static propTypes = {
+      data: PropTypes.object.isRequired,
+      drilldown: PropTypes.object,
+      options: PropTypes.object,
+      title: PropTypes.string
+    }
+
+    static defaultProps = {
+      drilldown: {},
+      options: {},
+      title: null
+    }
+
+    baseConfig = {
+      legend: {
+        position: 'bottom'
+      },
+      title: {
+        display: true,
+        fontSize: 18,
+        text: 'Default Area Chart'
```

Comment:
If this is generic, this title should probably read `'Default Chart'` or something